src/block/progress-bar/edit.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Internal dependencies
+ */
+import ProgressBarStyles from './style'
+import { DEFAULT_PERCENT } from '../../block-components/progress-bar/attributes'
+
+/**
+ * External dependencies
+ */
+import { InspectorTabs } from '~stackable/components'
+import {
+	BlockDiv,
+	Alignment,
+	Advanced,
+	Responsive,
+	MarginBottom,
+	Transform,
+	EffectsAnimations,
+	CustomAttributes,
+	CustomCSS,
+	ConditionalDisplay,
+	ProgressBar,
+	useGeneratedCss,
+	Typography,
+	getTypographyClasses,
+	getAlignmentClasses,
+} from '~stackable/block-components'
+import { version as VERSION } from 'stackable'
+import {
+	withBlockAttributeContext, withBlockWrapper, withQueryLoopContext,
+} from '~stackable/higher-order'
+import classnames from 'classnames'
+import striptags from 'striptags'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+
+const Edit = ( {
+	className, attributes,
+} ) => {
+	useGeneratedCss( attributes )
+
+	const blockAlignmentClass = getAlignmentClasses( attributes )
+	const textClasses = getTypographyClasses( attributes )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-progress-bar',
+		blockAlignmentClass,
+	] )
+
+	const textClassNames = classnames( [
+		'stk-progress-bar__inner-text',
+		textClasses,
+	] )
+
+	const barClassNames = classnames( 'stk-progress__bar', {
```

Comment:
I think the aim here is to have a common class for the **bar**, it might be better if we have 2 classes here: 
- `stk-progress-bar__bar` to satisfy BEM, and
- `stk-progress__bar` as the shared bar class